languages/ruby/exercises/concept/arrays/.meta/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "authors": [
+    {
+      "github_username": "pvcarrera",
+      "exercism_username": "pvcarrera"
```

Comment:
@ErikSchierboom is the creator of the concept exercise, I only ported it to Ruby, maybe @ErikSchierboom should be the one here, what do you think?

Reply:
@pvcarrera No this is fine. You are the author of the Ruby exercise and should be listed as such. The only thing you should add is the `forked_from` property (see [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#metaconfigjson)), which will indirectly attribute me as the original author of the forked exercise.